### PR TITLE
[Feat] Support Database Schema Builder

### PR DESCRIPTION
**Pull request by SonyPradana**

### CODE REVIEW

**Review by SonyPradana — COMMENTED** [review]
Without modifier `MyPDO::class` use overriding construction. Is better than creating new method, to prevent ambigus in features.

**Comment by SonyPradana** [comment]
Ready to next step
- [X] Database schema create and drop databas
- [ ] Database schema create, drop and flush table.

**Comment by anggerpradana** [comment]
Make sure pass phpstan min lvs 6

**Comment by anggerpradana** [comment]
Is nice if support serialize and deserialize

File: src/System/Database/MySchema/Table/Attributes/Constraint.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        return $this;
+    }
+
+    public function null(bool $null = true): self
+    {
+        return $this->notNull(!$null);
+    }
+
+    public function default(string $default): self
+    {
```
Comment (anggerpradana) [inline src/System/Database/MySchema/Table/Attributes/Constraint.php:56]: Add exceptions for default date, default date is `date()`.

**Comment by SonyPradana** [comment]
one step closser
[ ] support database flush table

File: src/System/Database/MySchema/Table/Attributes/Constraint.php (hunk 1 above)
Comment (anggerpradana) [reply src/System/Database/MySchema/Table/Attributes/Constraint.php:56]: Not really, or just another method to add raw constrain.

**Review by anggerpradana — APPROVED** [review]

**Pull request merged**